composer/callbacks/eval_output_logging_callback.py

```diff
@@ -104,6 +104,9 @@ def eval_batch_end(self, state: State, logger: Logger) -> None:
         self.rows.extend(rows)
 
     def eval_end(self, state: State, logger: Logger) -> None:
+        if not self.name or self.columns is None:
+            return
```

Comment:
self.name is set later in this function -- this check makes callback a noop?

Reply:
im pretty sure it gets set in eval_batch_end event, but only if you're processing valid batches for this callback. so you need to also skip the log table calls here if you're not processing valid batches for this callback. i'm gonna just revert this change and do separately though so i can get this pr in.